create_package/create_package.py

```diff
@@ -421,40 +419,48 @@ def matches(path, pattern):
 
     # I would LOVE to be able to use xz, but python's tarfile does not
```

Comment:
This comment looks irrelevant now (and should we consider using xz?).

Reply:
It has been for quite a while, we switched to python3 a long time ago :P However, i think we are good for now and can focus on more important stuff ; ) Thoughts @william-gr ?